syntax = "proto3";

///////////////////////////////////////////////////////////////////////////////
///////////////////////////////////////////////////////////////////////////////

package Reflector;

///////////////////////////////////////////////////////////////////////////////
///////////////////////////////////////////////////////////////////////////////

message AckRequest {
}
message AckResult {
}

service Service {
    rpc ack(AckRequest) returns(AckResult);
}

///////////////////////////////////////////////////////////////////////////////
///////////////////////////////////////////////////////////////////////////////
